service/handler_test.go

```diff
@@ -249,6 +253,8 @@ func TestHandlerServiceProcessPacketHandshake(t *testing.T) {
 			defer ctrl.Finish()
 
 			mockSerializer := serializemocks.NewMockSerializer(ctrl)
+			mockSerializer.EXPECT().GetName().AnyTimes()
```

Comment:
AnyTimes is not very good. To be sure it is called once, just drop the .AnyTimes()